sdk/storage/Azure.Storage.Blobs/src/Models/AppendBlobAppendBlockOptions.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Storage.Blobs.Models
+{
+    /// <summary>
+    /// Optional paratmers for appending a block.
```

Comment:
Typo, also we can add a reference here to the `AppendBlobClient.AppendBlockAsync()` which takes this overload.